README.md

```diff
@@ -245,3 +245,12 @@ flags:
 ```
 
 This will remap the "environment" Terraform variable to `--env` on the astro command line. You can also specify a description that will show up in the `--help` text.
+
+## Contributing to astro
+
+**Releasing a new version**
```

Comment:
This reads more easily as a list:

```
**Releasing a new version**

1. Make sure the CHANGELOG is updated
2. Create and push a new tag named `x.x.x` (following [semver](https://semver.org/))
3. The [release action](https://github.com/uber/astro/actions?workflow=release) will automatically build and upload binaries
```

Also we can add:

```
**Making and testing changes to the release process**

1. Make your changes to the [release GitHub action workflow file](https://github.com/uber/astro/actions?workflow=release) and push it to a branch
2. Create a new tag pointing to the tip of the branch (name it: `0.0.0-dev`)
3. Push the tag to trigger the action. The action will use the updated config.
4. Delete the tag after testing (`git tag -d 0.0.0-dev; git push --delete origin 0.0.0-dev`)
```

Disclaimer: I haven't tested the instructions for "Testing the release process" so we should verify that it works before landing (e.g. that the tag name 0.0.0-dev works).

Reply:
Also I'm starting to think these runbooks should be in the wiki and we should link to there from the README instead of making the README really long.